pkg/configurator/client.go

```diff
@@ -24,46 +24,46 @@ const (
 )
 
 const (
-	// PermissiveTrafficPolicyModeKey is the key name used for permissive mode in the ConfigMap
+	// PermissiveTrafficPolicyModeKey is the key name used for permissive mode in the MeshConfig
 	PermissiveTrafficPolicyModeKey = "permissive_traffic_policy_mode"
 
-	// egressKey is the key name used for egress in the ConfigMap
+	// egressKey is the key name used for egress in the MeshConfig
 	egressKey = "egress"
 
-	// enableDebugServer is the key name used for the debug server in the ConfigMap
+	// enableDebugServer is the key name used for the debug server in the MeshConfig
```

Comment:
These comments are not accurate. The key names are still referring to the older ConfigMap keys. Ex. `permissive_traffic_policy_mode -> enablePermissiveTrafficPolicyMode`

Reply:
all these keys are only used in testing (client_test only). I think it would be in line to also get rid of them in a separate commit, by a caritative soul that wants to handle this unholy burden.

Reply:
Yes, we can get rid of keys all together which will be handled in a separate commit, it is tracked by this issue : https://github.com/openservicemesh/osm/issues/3268